crates/payload/basic/src/lib.rs

```diff
@@ -57,6 +57,9 @@ use tracing::{debug, trace};
 
 mod metrics;
 
+#[cfg(not(feature = "optimism"))]
+pub type EthereumPayloadBuilder = ();
```

Comment:
I'd like to not use a type alias here and instead a zero sized new struct for this